path_config.go

```diff
@@ -41,6 +49,14 @@ func pathConfig(b *jwtAuthBackend) *framework.Path {
 					Sensitive: true,
 				},
 			},
+			"oidc_response_mode": {
+				Type:        framework.TypeString,
+				Description: "The response mode to be used in the OAuth2 request. Allowed values are 'query' and 'form_post'.",
+			},
+			"oidc_response_types": {
+				Type:        framework.TypeCommaStringSlice,
+				Description: "The response typess to request. Allowed values are 'code' and 'id_token'. Defaults to 'code'.",
```

Comment:
We also support the multiple response types yeah? https://openid.net/specs/oauth-v2-multiple-response-types-1_0.html#Combinations
So allowed values may be things like 'code id_token' 'id_token code' .. ('token id_token code' ??)

Reply:
We do support multiple types, but they're provided individually (note the type `TypeCommaStringSlice`). So `oidc_response_types=code,id_token` or even `oidc_response_types=code oidc_response_types=id_token`. Order doesn't matter as we combine the segments into a string later.

Reply:
Fixed `typess`

Reply:
Ah okay, thanks for clarification. (Feel free to resolve)